### PR TITLE
Resizable splits, docked panels, and `attn open` for markdown

### PR DESCRIPTION
**Pull request by victorarias**
Two related pieces of workspace-layout work that build on each other.

## 1. Resizable splits + docked panels (`1bc85752`)
- Drag the divider between any two workspace panes to resize; the ratio is daemon-owned and persists across reconnects/restarts and other clients.
- Panels can dock directly into the layout as a real, resizable pane instead of floating. Drag a panel's title bar onto a terminal to re-dock on that edge, or drop between two terminals to slot in between. Placement/size persist like splits.
- The slide-in diff/review/git-status/PR overlays are intentionally unchanged.

## 2. Open markdown in a docked panel via `attn open` (`ea913216`)
- `attn open <file.md>` docks a markdown panel next to a session and renders the file.
- **Content is daemon-owned, not frontend-read**: the panel subscribes with `workspace_panel_content_get`; the daemon replies and broadcasts `workspace_panel_content` to all clients.
- **Live reload** via a poll-based watcher (no fsnotify — robust to atomic saves) that re-derives its watch set from stored layouts each tick.
- File path rides the layout as opaque `panel_params`, so reopening the app or another client reproduces the same document.
- Session resolution: `--session` > `ATTN_SESSION_ID` > the daemon's currently-selected session.
- The bundled attn agent skill documents `attn open` so agents can show a plan/report as rendered markdown.
- Removed the empty "show markdown panel" toggle; a panel only exists once it points at a real file. Drag-to-dock unchanged.
- Protocol bumped to 76.

## Verification
- `go test ./...` green, `go vet` clean, gofmt clean.
- Frontend `tsc --noEmit` clean, 474 unit tests pass.
- Manual end-to-end against the real dev daemon: `attn open` docks with the right path; content served over WS; editing the file broadcasts updated content (live reload); workspace screenshot shows the rendered panel.

## Design note
See `docs/decisions/2026-05-31-docked-panels.md` for the daemon-owned-layout rationale (and why polling over fsnotify).

### CODE REVIEW

**Review by figgyster — DISMISSED** [review]
This is a real contract miss, not a taste thing: the documented `attn open <file.md> [--session <id>]` form doesn't actually honor `--session` when it appears after the path.

`cmd/attn/main.go` uses Go's `flag.FlagSet.Parse(os.Args[2:])`, and the standard flag parser stops at the first non-flag argument. So `attn open README.md --session other` opens against `ATTN_SESSION_ID`/the selected session instead of `other`, while still succeeding. That's the kind of CLI lie that sends users spelunking through the wrong workspace with a flashlight and bad intentions.

Required before merge:
- Accept the documented trailing `--session` form, or change the usage/docs and tests to the flag-before-path form.
- Add a CLI parsing regression for `attn open README.md --session <id>` so this doesn't quietly come back wearing a fake mustache.

I verified the rest of the reviewed surface with `git diff --check`, focused Go tests for `cmd/attn`, daemon/workspace/protocol/agent packages, and frontend `tsc --noEmit` plus focused Vitest coverage. Those were green; this CLI parsing bug is the blocker.

File: cmd/attn/main.go [hunk 1]
```diff
@@ -342,6 +345,39 @@ func runList() {
...
 }
 
+// runOpen handles `attn open <file.md> [--session <id>]`, docking a
+// live-reloading markdown panel into a workspace. The session defaults to
+// ATTN_SESSION_ID (set inside attn-managed agents), then the daemon's currently
+// selected session.
+func runOpen() {
+	fs := flag.NewFlagSet("open", flag.ExitOnError)
+	sessionID := fs.String("session", "", "session id (defaults to ATTN_SESSION_ID, then the selected session)")
+	_ = fs.Parse(os.Args[2:])
```
[the remark this answers is not in the document]
Comment (victorarias) [reply cmd/attn/main.go]: Fixed in 6331d8dc (on behalf of Victor). `attn open` now parses interspersed flags and the path positional, so `--session` is honored before *or* after the path (and the `--session=<id>` form). Added a `parseOpenArgs` regression test covering both orderings plus missing-path, extra-positional, and unknown-flag cases. 👍

**Review by figgyster — DISMISSED** [review]
The CLI contract miss is fixed: `attn open README.md --session other` now parses the trailing flag instead of quietly docking into the wrong session, and the regression tests cover both argument orders plus the ugly edge cases.

I also checked the workspace layout/panel path and the protocol surfaces for the docked markdown flow. Local focused checks passed:
- `git diff --check origin/main...HEAD`
- `go test ./cmd/attn ./internal/workspacelayout ./internal/daemon ./internal/store ./internal/protocol ./internal/agent`
- `pnpm --dir app exec vitest run App.worktreeCleanup.test.tsx useDaemonSocket.test.tsx workspace.test.ts`

No blockers from me. CI was still finishing when I reviewed, but the contract issue I blocked on is resolved.

**Review by figgyster — DISMISSED** [review]
This looks safe to merge. The `attn open` parser now honors the documented trailing `--session` form, and the docked markdown panel path stays daemon-owned through layout params/content broadcasts instead of sneaking file reads into the frontend.

I ran `git diff --check`, `go test ./...`, `pnpm --dir app test`, and `pnpm --dir app exec tsc --noEmit` locally. No blockers from me.

**Review by figgyster — APPROVED** [review]
This looks safe to merge. The `attn open` parser now honors the documented trailing `--session` form, and the docked markdown/panel layout path has the right daemon-owned persistence shape.

Focused local checks passed (`git diff --check`, the relevant Go packages, and the workspace/socket frontend tests). No blockers from me.

**Review by figgyster — APPROVED** [review]
This looks safe to merge. The follow-up tightened the docked markdown path in the right direction: content stays daemon-owned, local Markdown targets go through a guarded Tauri command, and remote workspace panels don’t get to open arbitrary local files. That’s the kind of paranoia I like in UI code that renders repository text.

Focused checks passed locally: `git diff --check`, targeted Go packages, targeted workspace/panel frontend tests, and `tsc --noEmit`. GitHub’s Frontend/Tauri jobs were still running when I posted this; Daemon/Changes/Linux daemon were green.

**Review by figgyster — APPROVED** [review]
This still looks safe to merge. The follow-up adds the `attn presence` probe and help text without disturbing the docked markdown contract, and the presence env is wired at the PTY spawn boundary where it belongs.

I rechecked the incremental change with `git diff --check` and `go test ./...`. GitHub CI was still finishing on this head, but no blockers from me.
